build.gradle.kts

```
@@ -23,13 +23,21 @@ plugins.withType<JavaPlugin>().configureEach {
 group = "org.junit-pioneer"
 description = "JUnit 5 Extension Pack"
 
-val modularBuild = findProperty("modularBuild") != null;
+val modularBuild : String by project;
+val experimentalJavaVersion : String? by project;
+val experimentalBuild: Boolean = experimentalJavaVersion?.isNotEmpty()?: false
 
 java {
-	if(modularBuild) {
-		sourceCompatibility = JavaVersion.VERSION_11
+	if(experimentalBuild) {
+		toolchain {
+			languageVersion.set(JavaLanguageVersion.of(experimentalJavaVersion))
+		}
 	} else {
-		sourceCompatibility = JavaVersion.VERSION_1_8
+		if(modularBuild.toBoolean()) {
```

```suggestion
		if (modularBuild.toBoolean()) {
```

(We need a Spotless config for Kotlin, just like we used to have with Groovy: #484)